src/applications/disability-benefits/all-claims/utils.jsx

```diff
@@ -798,3 +804,9 @@ export const activeServicePeriods = formData =>
   _.get('serviceInformation.servicePeriods', formData, []).filter(
     sp => !sp.dateRange.to || moment(sp.dateRange.to).isAfter(moment()),
   );
+
+/**
+ * SiP strips out the - in the uuid, so to make it SiP-safe, strip
+ * them out at generation
+ */
+export const sippableId = () => uuid().replace(/-/g, '');
```

Comment:
This is arguably the wrong place for it, but since we're only using it in here, I figured it made the most sense to only include it in this bundle